.github/workflows/verify.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
         run: ./gradlew checkstyleMain checkstyleTest
 
   Dependency-Check:
+    if: ${{ inputs.skip-dependency-check != 'true' }}
```

Comment:
no need for this, dependency-check won't run on scheduled workflows anyway (ref. https://github.com/eclipse-edc/.github/blob/dfb2f1c61348c41d35003b7ab04fc1dd33655077/.github/workflows/dependency-check.yml#L28)